server/broadcast.go

```diff
@@ -346,12 +356,28 @@ func transcodeSegment(cxn *rtmpConnection, seg *stream.HLSSegment, name string,
 
 	// send segment to the orchestrator
 	glog.V(common.DEBUG).Infof("Submitting segment nonce=%d manifestID=%s seqNo=%d orch=%s", nonce, cxn.mid, seg.SeqNo, sess.OrchestratorInfo.Transcoder)
+	if sess.Sender != nil {
+		err := sess.Sender.ValidateTicketParams(pmTicketParams(sess.OrchestratorInfo.TicketParams))
+		if err != nil {
+			if err != pm.ErrTicketParamsExpired {
+				return nil, err
+			}
 
+			glog.V(common.VERBOSE).Infof("ticketparams expired, refreshing for orchestrator=%v", sess.OrchestratorInfo.Transcoder)
```

Comment:
We use "ticket params" instead of "ticketparams" elsewhere ([example](https://github.com/livepeer/go-livepeer/blob/d3f536acf007f8a98edf76f007f2e6ea5517987d/discovery/db_discovery.go#L102)) so I think we should stick with the former for consistency.

Reply:
fixed all these small change requests in this file in 774341f6e04e79aade04aaa6b7e98a07fd26618d